jekyll/_data/sidenav.yml

```diff
@@ -352,7 +352,7 @@ en:
       link: 2.0/server-3-overview
       children:
         - name: v3.0 Overview
-          link: 2.0/CircleCI-Server-3.0.0-Overview.pdf
+          link: 2.0/CircleCI-Server-{serverversion}-Overview.pdf
         - name: v3.0 Installation Guide
           link: 2.0/CircleCI-Server-3.0.0-Installation-Guide.pdf
```

Comment:
does this need to be updated too or does it stay at 3.0.0? otherwise lgtm!

Reply:
if yes, then also needs updating on line 359

Reply:
nah, {serverversion} is a placeholder for the variable vallue :) it will be populated with 3.0.1